walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,88 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors; 
+import java.util.stream.Stream; 
 
 public final class FindMeetingQuery {
+  /**
+   * Method for retrieving all possible times for the request based on all attendee's
+   * events and availabilty.
+   * @return Collection of TimeRanges that show when the requested meeting can occur.
+   */
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
```

Comment:
Just provide another way to solve the problem. After we get sortedEventTimeRanges, we sort them, then merge them. Then find out all the available time slots. Then iterate through the available time slots to find time ranges longer than MeetingRequest. 
Your solution is good. I just want to share some thoughts I had.